packages/react-heartwood-components/src/components/Core/components/SaveBar/SaveBar-story.js

```diff
@@ -45,7 +34,6 @@ stories
 			isDiscardDisabled={boolean('isDiscardDisabled', false)}
 			onDiscard={() => console.log('DISCARD')}
 			onSave={() => console.log('SAVE')}
-			isSaving={true}
```

Comment:
same with the button story, is there a better way to include this such that it'll come up in the knobs?